polaris-react/src/components/PositionedOverlay/PositionedOverlay.module.scss

```diff
@@ -1,6 +1,17 @@
+:root {
+  // stylelint-disable polaris/conventions/polaris/custom-property-allowed-list -- PositionedOverlay CSS Properties; Supports consuming component styles tying into changes in order to animate without hacky JavaScript timers and forcing reflow
+  --pc-positioned-overlay-top: initial;
+  --pc-positioned-overlay-bottom: initial;
+  // stylelint-enable
+}
+
```

Comment:
CSS Custom Properties have a default value of `initial` when not set, so defining these on `:root` is the same as not defining them at all.

```suggestion
```